src/js/initializers/initConnectionParams.ts

```diff
@@ -20,14 +22,12 @@ const setupConnection = (host, port) => (dispatch, _, {globalDispatch}) => {
 }
 
 export default function(store: Store) {
-  const {host, port, id} = getUrlSearchParams()
+  const {id} = getUrlSearchParams()
   global.windowId = id
 
   const existingConnection = Current.getConnection(store.getState())
 
-  if (host && port) {
-    store.dispatch(setupConnection(host, port))
```

Comment:
here was another bug: searching from a detail window back to a search window (via context menu) was  resetting the connection's name back to it's default (host:port). Using these host and port params isn't necessary anymore though